igvc_gazebo/launch/simulation.launch

```diff
@@ -18,6 +18,7 @@
 
     <param name="/use_sim_time" value="true"/>
     <param name="/waypoint_file_path" value="$(arg waypoints)" />
+    <param name="noise" type="double" value="0.0"/>
```

Comment:
I think you should make this parameter a bit more descriptive. Does this represent the standard deviation / variance of the noise? On what values (x, y, z, heading) does it get applied to?